lib/url.c

```diff
@@ -1521,6 +1521,16 @@ CURLcode Curl_setopt(struct SessionHandle *data, CURLoption option,
                        va_arg(param, char *));
     break;
 
+#ifdef HAVE_GSS_ACQUIRE_CRED_FROM
+  case CURLOPT_KRB_CCACHE:
```

Comment:
Make sure that you retain the sematics `KRB5CCNAME` and/or `gss_acquire_cred_from` imply.